### PR TITLE
Update README with widget instructions

### PR DESCRIPTION
**Pull request by aperala**
First draft ...

### CODE REVIEW

**Comment by acabunoc** [comment]
Nice @aperala! Great start. 

Can you add a bit of text describing what the widget does and maybe a screenshot?

Once that's in, I'm happy to merge this :smiley:

**Comment by aperala** [comment]
Thanks, @acabunoc Would you be able to upload a screenshot that I can add? I actually have not got this working on my end.

**Comment by acabunoc** [comment]
Oh dear, what are you seeing if this doesn't work for you?

**Comment by Alicole** [comment]
@acabunoc -- for following link, https://badges.mozillascience.org/papers/10.1186/2047-217X-3-18/badges?pretty=true , we're currently experiencing:

ERROR

The request could not be satisfied.

CloudFront attempted to establish a connection with the origin, but either the attempt failed or the origin closed the connection. 
Generated by cloudfront (CloudFront)
Request ID: c11ysRT2jxPhF-IFXXRrOwmE3Wli6y3gFXmMAofOF2lW5PiSduAosw==

**Comment by acabunoc** [comment]
@Alicole - restarted our badgekit!

File: README.md [hunk 1]
```diff
@@ -41,6 +41,34 @@ Ask @acabunoc for ones marked `###########`. Our custom BadgeKit API code can be
...
 
+### Using the Widget
+1. To use the widget on your own site, include a `<div>` with your custom class in your view file, for example:
+    `<div class="my-container"></div>`
+
+2. Add `<script src="https://badges.mozillascience.org/widgets/paper-badger-widget.js"></script>` above the closing `<body>` tag. 
+
+3. In your scripts, include your custom class name as the value for the "container-class" key, for example:
+
+```
```
Comment (acabunoc) [inline README.md]: specify html after the <code>```</code> to get the pretty markup

    ```html
     <p>nicer html display here</p>
    ```

File: README.md [hunk 2]
```diff
@@ -41,6 +41,40 @@ Ask @acabunoc for ones marked `###########`. Our custom BadgeKit API code can be
 
 5. Run `npm start`, and open up `http://localhost:5000/` in your favourite web browser!
 
+### Using the Widget
+Scholars earn badges for their specific contributions to a scientific paper. A scholar who worked on investigation earns a prestigious investigation badge for that paper.
```
Comment (acabunoc) [inline README.md]: Can we tweak the language a bit to be more consistent?

Scholar -> researcher
scientific paper -> academic paper

**Comment by acabunoc** [comment]
Looks great @aperala - just the one tweak, then it's good to go :)

**Comment by acabunoc** [comment]
:+1: 

Merging!

yay

**Pull request merged**